matrix.go

```diff
@@ -131,13 +130,16 @@ func (m *Matrix) Inverse() (*Matrix, error) {
 	return inverse, nil
 }
 
-func GetI(c int) *Matrix {
+func GetI(c int, context *Int) *Matrix {
```

Comment:
I'm confused by what this function does. I think a comment should be added (I will then review it better)

Reply:
It creates an identity matrix of a given size: c is the size (identity matrix is always square).

Reply:
I'll add a comment

Reply:
added comment